Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FrameworkElementExtensions/FrameworkElementExtensionsCode.bind

```diff
@@ -31,30 +31,28 @@
       <controls:GridSplitter
         Width="11"
         Background="{ThemeResource Brush-Grey-04}"
-        GripperCursor="Default"
+        GripperCursor="Arrow"
         HorizontalAlignment="Left"
         Grid.Column="1"
         ResizeDirection="Auto"
         ResizeBehavior="BasedOnAlignment"
         CursorBehavior="ChangeOnGripperHover"
-        GripperForeground="{ThemeResource Brush-Alt}">
+        Foreground="{ThemeResource Brush-Alt}">
       </controls:GridSplitter>
 
       <!--Row Grid Splitter-->
       <controls:GridSplitter
         Grid.Row="1"
         Background="{ThemeResource Brush-Grey-04}" Height="11"
         HorizontalAlignment="Stretch">
-        <controls:GridSplitter.Element>
+        <controls:GridSplitter.Content>
```

Comment:
**should** being the keyword; getting a runtime error at the moment when loading up this page w/ the modifications:  
![image](https://user-images.githubusercontent.com/86266896/152619803-1b68b49c-6034-4f7e-ab6b-44f9e8248832.png)

Reply:
Applying this attribute to GridSplitter seems to fix this issue:  `[ContentProperty(Name = nameof(Content))]`

Reply:
Yes, the ContentPropertyAttribute tells the XAML compiler which property to stick unmarked children in. The question is if we can specify that attribute once on the base-class vs. both on GridSplitter/ContentSizer.

Reply:
> It may be good to break-down the API sets between GridSplitter and ContentSizer and then what we want each of them to look like and the base class. 

Take a look at this new issue that has just spawned:  https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/issues/606